src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProject.java

```diff
@@ -128,6 +130,14 @@ public void setEndDate(Instant endDate) {
         this.endDate = endDate;
     }
 
+    public List<Object> getGrant() {
```

Comment:
Is the type of grant really Object?

Reply:
It is just a placeholder until we actually implement this field in later issue